src/Bicep.Cli.IntegrationTests/packages.lock.json

```diff
@@ -33,6 +33,16 @@
           "Microsoft.TestPlatform.TestHost": "16.10.0"
         }
       },
+      "Microsoft.SourceLink.GitHub": {
```

Comment:
Do we need the package for test projects though?

Reply:
Technically not, but I think it's going to be pulled in as a transitive dependency anyway for any projects taking a dependency on Bicep.Core. I could definitely be more precise in targeting just projects building nugets, but just went for the simplest approach.

Reply:
Makes sense!

Reply:
Btw, checked out the generated nupkg and dependencies look good:
![image](https://user-images.githubusercontent.com/22460039/152080596-39a3cfb3-ba7f-423b-ad21-97cfd037f3dd.png)